lib/gym/runner.rb

```diff
@@ -1,6 +1,7 @@
 require 'pty'
 require 'open3'
 require 'fileutils'
+require 'shellwords'
```

Comment:
Good call, AFAIK it's in the standard library and doesn't need to be added. We use it already in `gym`